R/loo_compare.R

```diff
@@ -42,10 +42,19 @@
 #'   distribution, a practice derived for Gaussian linear models or
 #'   asymptotically, and which only applies to nested models in any case.
 #'
+#'   If more than \eqn{11} models are compared, then the worst model by elpd is
+#'   taken as the baseline model, and the risk of the difference in predictive
```

Comment:
I'm slightly confused by this because the printed `loo_compare` results will still show the model with the _best_ ELPD as the baseline (i.e. all `elpd_diff` are relative to that one), right? 

If that's still true, then I think we should rephrase this to indicate that this is just happening in an internal check and doesn't affect which model is considered as the baseline for computing `elpd_diff` in the output. Does that make sense? Or am I just confused (totally possible)?

Reply:
I have now updated the docstring, let me know if it is now clearer 🙏